.github/workflows/staging-build-tar.yml

```diff
@@ -23,755 +23,111 @@ jobs:
-      - name: Checkout SQL
-        uses: actions/checkout@v1
-        with:
-          repository: opendistro-for-elasticsearch/sql
-          ref: ${{env.p_tag_sql}}
-      - name: IT
-        run: |
-          ./release-tools/scripts/setup_runners_service.sh zip --es
-          export PATH=$JAVA_HOME:$PATH; cd $GITHUB_WORKSPACE/../sql; pwd
-          ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername=es-integrationtest -Dhttps=true -Duser=admin -Dpassword=admin
-  
-  Test-AD:
-    needs: [build-es-artifacts]
-    runs-on: ubuntu-18.04
-    name: Test-AD
-    strategy:
-      fail-fast: false
-      matrix:
-        java: [14]
-    steps:
-      - name: Set up AWS Cred
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
-          aws-region: us-east-1
-      - name: Setup Java
-        uses: actions/setup-java@v1
-        with:
-          java-version: ${{ matrix.java }}
-      - uses: actions/checkout@v1
-      - name: Required Packages
-        run: ./release-tools/scripts/required_packages.sh
-      - name: Retrieve plugin tags
-        run:  echo "p_tag_ad=$(release-tools/scripts/plugin_tag.sh opendistro-for-elasticsearch/anomaly-detection)" >> $GITHUB_ENV
-      - name: Checkout anomaly-detection
-        uses: actions/checkout@v1
-        with:
-          repository: opendistro-for-elasticsearch/anomaly-detection
-          ref: ${{env.p_tag_ad}}
-      - name: IT
-        run: |
-          ./release-tools/scripts/setup_runners_service.sh zip --es
-          export PATH=$JAVA_HOME:$PATH; cd $GITHUB_WORKSPACE/../anomaly-detection; pwd
-          ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="es-integrationtest" -Dhttps=true -Duser=admin -Dpassword=admin
-
-  Test-IM:
-    needs: [build-es-artifacts]
+  build-es-artifacts-x64:
+    needs: [plugin-availability]
+    name: Build ES Artifacts X64
     runs-on: [ubuntu-18.04]
-    name: Test-IM
-    strategy:
-      fail-fast: false
-      matrix:
-        java: [14]
     steps:
       - uses: actions/checkout@v1
-      - name: Set up AWS Cred
+      - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
           aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
       - name: Required Packages
         run: ./release-tools/scripts/required_packages.sh
-      - name: Retrieve plugin tags
-        run:  echo "p_tag_im=$(release-tools/scripts/plugin_tag.sh opendistro-for-elasticsearch/index-management)" >> $GITHUB_ENV
+      - name: Build tar
+        run: set -u; ./elasticsearch/linux_distributions/opendistro-tar-build.sh linux x64
```

Comment:
Please remove "linux x64" as they are default.

Reply:
It does not hurt for us to specify these parameters, in case someone change the script and mess up these values.